apps/server/lib/lexical/server/code_intelligence/completion/translations/struct.ex

```diff
@@ -69,21 +69,25 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Struct do
           prefix_end
 
         {:struct, typed_module_name} ->
-          edit_begin =
-            case left_offset_of(typed_module_name, ?.) do
-              {:ok, offset} ->
-                env.position.character - offset
+          edit_begin_of_struct(env, typed_module_name)
 
-              :error ->
-                env.position.character - length(typed_module_name)
-            end
-
-          edit_begin
+        {:local_or_var, [?_ | _rest] = typed} ->
```

Comment:
you sure a single underscore is enough to decide that this is `__MODULE__`?

Reply:
Yes, and only the call from `Translations.Macro` will arrive here.

Reply:
ok.